pyriemann/utils/mean.py

```diff
@@ -577,18 +577,22 @@ def mean_covariance(covmats, metric='riemann', sample_weight=None, *args):
         'wasserstein', or a callable function.
     sample_weight : None | ndarray, shape (n_matrices,), default=None
         Weights for each matrix. If None, it uses equal weights.
-    args : list of params
-        The arguments passed to the sub function.
+    kwargs : list of params
+        The keyword arguments passed to the sub function.
```

Comment:
```suggestion
    **kwargs : dict
        The keyword arguments passed to the sub function.
```

it the numpydoc way

See https://numpydoc.readthedocs.io/en/latest/example.html